src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/external/UserRolesController.kt

```diff
@@ -221,7 +222,20 @@ class UserRolesController(
       SecurityRequirement(name = "ROLE_MAINTAIN_IMS_USERS"),
     ],
   )
-  @StandardApiResponses
+  @ApiResponses(
```

Comment:
we have a standard set of responses for the endpoint would this not be better going in there - https://github.com/ministryofjustice/hmpps-manage-users-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/swagger/StandardApiResponses.kt